docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,68 @@
+  * Using `<br>` might be necessary only in lists of items
+  
+## When to Use Emphasis
+  
+Whether a user knows it or not, they will come to recognize styles that are consistent throughout the text. This way, their eyes can jump to a part of the text that addresses their issue, if they don’t need to read the whole thing. Follow these rules on using emphasis:
+
+* *Italicize* when referring to a specific place on the screen
+  * e.g. in the Selection Filters section
+  * e.g. in the Metadata column of the table
+  * e.g. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * e.g. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * e.g. In the **Imaging Browser** module
+    * e.g. Click the **Upload** tab
+  * You can also use bold when defining features in a list, to separate the term being defined from the definition itself, e.g. when describing columns in a table:<br>
+     **Column heading 1**: description<br>
+     **Column heading 2**: description<br>
+     Etc.
+     
+## Tone
+
+LORIS is used by scientific community members worldwide, which means it's possible that most users are non-native English speakers. Therefore, it is important that LORIS documentation (help text, setup guides, etc.) be written in a way that is quickly comprehensible by people with many different levels of English language and expression.
+
+This means using more direct language and tone to get information across to the user in the least amount of time, with the least amount of friction. The quicker a user understands something, the happier they are. Easily understandable documentation also builds trust in the software, in addition to decreasing the amount of support questions we get.
+
+* Use second person point of view rather than third
+  * e.g. “This module allows *you* to browse…” (rather than “This module allows *users* to browse…”)
+* Keep it concise. Avoid unnecessary information.
+  * Try not to be repetitive. 
+  * e.g. the Help text for the Imaging Browser module begins with the title Imaging Browser. You don’t need to start text with “In the Imaging Browser module...”, because the user already knows the module based on the title. Instead, keep it short and say “In this module…”
+* Assume the user is tech-savvy enough to know about features that are intuitive or common across modern platforms, so you don’t need to repeat in every Help text
+  * e.g. clicking the column headers to re-sort a table
+  * e.g. clicking a Download button to download something
+  * e.g. navigate pages of a table using the arrows provided
+* Use Plain Language
+  * Avoid words that you wouldn't use in everyday speech
+* Where relevant, introduce an abbreviation once and then use that abbreviation for the rest of the copy in that help text (e.g. introduce Quality Control = QC in a help text title, then only call it QC throughout the text)
```

Comment:
```suggestion
* Where relevant introduce an abbreviation once and then use that abbreviation for the rest of the copy in that help text (e.g. introduce Quality Control = QC in a help text title, then only call it QC throughout the text)
```

I don't think this comma is necessary either.

Reply:
I think the comma is better than no comma. 

Maybe the "Where relevant" can be removed from the beginning of the sentence though. The sentence stands on its own without it.

Or maybe "When introducing an abbreviation, use it once and then...'"

Reply:
I don't feel strongly about this particular comma.

Reply:
Nor I